### PR TITLE
Tc3 skincare routine score

### PR DESCRIPTION
**Pull request by hernandezvictoria**
## Description

- computation for skincare routine score
- testing helper functions and skincare routine score
- made sure to extract sub-computations into helper functions
- added comprehensive docstrings for increased readability

## Milestones
- TC3

## Resources
- N/A

## Test Plan
- See skincare-routine.test.js for lots of testing and lots of edge case considerations!

### CODE REVIEW

File: backend/data-processing/import-ingredients.js [hunk 1]
```diff
@@ -0,0 +1,148 @@
...
+const {
+    SkinTypes,
+    SkinConcerns,
+    HarshIngredientTypes,
+} = require("../enums.js");
+
+fs.createReadStream("ingredients.csv")
+    .pipe(parse({ columns: true, trim: true }))
+    .on("data", async (row) => {
+        const getConcernsArray = (str) => {
```
Comment (teaenshin) [inline backend/data-processing/import-ingredients.js:14]: nit: can we use a map from skin concerns to keywords like `{[SkinConcerns.ACNE]: ['acne', 'oiliness', 'antimicrobial']...}` and iterate over the map + use Array.some to make this series of if statements more readable/extendable?

**Review by teaenshin — APPROVED** [review]
